dftimewolf/lib/processors/gcp_cloud_resource_tree_helper.py

```diff
@@ -292,22 +292,24 @@ def __str__(self) -> str:
     for entry in result:
       resource: Optional[Resource] = entry.get('resource_object')
       if resource:
-
         output.write(
-            f'{resource.id if ("-" not in resource.id) else "":<20s}|' +
-            f'{resource.type:<18s}|' +
-            f"""{resource.creation_timestamp.astimezone(
-                         datetime.timezone.utc).strftime("%Y-%m-%d %H:%M:%S")
-                         if resource.creation_timestamp else "":<20s}|""" +
-            f'{resource.created_by:<25s}|' +
-            f'{resource.creator_ip_address:<18s}|' +
-            f"""{resource.deletion_timestamp.astimezone(
-                         datetime.timezone.utc).strftime("%Y-%m-%d %H:%M:%S")
-                         if resource.deletion_timestamp else "":<20s}|""" +
-            f'{resource.deleted_by:<25s}|' +
-            f'{resource.deleter_ip_address:<18s}|' +
-            f'{entry.get("tree"):<100s}\n')
-
+          # pylint: disable=line-too-long
+          # If the resource.id contains a '-' then its a fake one we created for tracking so we are not displaying it in the output
+          f'{resource.id if ("-" not in resource.id and resource.id) else "":<20s}|'
```

Comment:
why not make this condition:
`resource.id and "-" not in resource.id`
thus the existence of resource.id gets evaluated first and the condition fails if it doesn't exist, i.e. avoid errors if resource.id is null

Reply:
Ya it should be reversed. I fixed that but maybe didn't save it.